site/content/tutorial/05-events/03-event-modifiers/text.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Event modifiers
+---
+
+DOM event handlers can have *modifiers* that alter their behaviour. For example, a handler with a `once` modifier will only run a single time:
+
+```html
+<script>
+	function handleClick() {
+		alert('no more alerts')
+	}
+</script>
+
+<button on:click|once={handleClick}>
+	Click me
+</button>
+```
+
+The full list of modifiers:
+
+* `preventDefault` — calls `event.preventDefault()` before running the handler. Useful for e.g. client-side form handling
+* `stopPropagation` — calls `event.stopPropagation()`, preventing the event reaching the next element
+* `passive` — improves scrolling performance on touch/wheel events (Svelte will add it automatically where it's safe to do so)
+* `capture` — fires the handler during the *capture* phase instead of the *bubbling* phase
+* `once`
```

Comment:
I know you just mentioned what `once` does up higher, but it does look a little weird to me not having some sort of a short description for it here.

Reply:
updated this